aws_lambda_powertools/utilities/validation/base.py

```diff
@@ -8,7 +8,9 @@
 logger = logging.getLogger(__name__)
 
 
-def validate_data_against_schema(data: Union[Dict, str], schema: Dict, formats: Optional[Dict] = None):
+def validate_data_against_schema(
+    data: Union[Dict, str], schema: Dict, formats: Optional[Dict] = None, handlers: Optional[Dict] = None
+):
```

Comment:
Hello @danieljnchen! I'm looking at the [Validation API](https://horejsek.github.io/python-fastjsonschema/#fastjsonschema.validate) and it seems to accept other parameters like `use_default`, `use_formats` and they may add others in the future. So what do you think about adding `**kwargs` and propagating that to the `validate` function?

Reply:
`**provider_options` so it's more explicit. ideally, we'd encapsulate all that in a `provider`, so a customer could swap `fastjsonschema` with `jsonschema` and whatnot too; it'd be too much work for Daniel, so just having an explicit kwargs name + docs help

Reply:
Makes sense, and thanks for the feedback! I'll add some `**provider_options` to make sure we can pass other parameters in the future.

Reply:
Alright, thanks for your patience! I've added the `**provider_options` arg to `validate_data_against_schema` and fixed the tests that were failing based on what I could see in the CI pipeline. Still haven't gotten tests running in my dev environment so I may have missed something ;)

Reply:
Hello @danieljnchen! Thanks a lot for addressing our feedback and updating this PR! I see some items failing in our CI, but I'm sure it's because you couldn't run local tests. I know you're blocked because the environment we create in gitpod is broken.

We are fixing this and we can come back with the review of this PR next week, ok?

Reply:
No problem, thank you for your feedback and help with the environment issue! Happy to pick this up next week :)

Reply:
Hello @danieljnchen! We have good news. We fixed the Gitpod environment and you can use it now. You may need to rebuild your existing workspace with this new configuration.

https://github.com/aws-powertools/powertools-lambda-python/pull/4532

Reply:
> **provider_options so it's more explicit. ideally, we'd encapsulate all that in a provider, so a customer could swap fastjsonschema with jsonschema and whatnot too; it'd be too much work for Daniel, so just having an explicit kwargs name + docs help

Hello @heitorlessa! I'm going to open an issue to implement this because we need to refactor a lot of things to make this idea work.